include/style/article.styl

```diff
@@ -148,3 +148,17 @@ article
         
         a
             color: inherit
+
+a
+    &.article-nav-prev
+        span
+            text-align: left
+            width: 100%;
```

Comment:
What about

```diff
- width: 100%
+ flex-shrink: 1
```

Before:
![Screen Shot 2020-09-02 at 15 16 58](https://user-images.githubusercontent.com/8849362/92002886-e46fcf00-ed2f-11ea-8641-025177950471.png)


After:
![Screen Shot 2020-09-02 at 15 17 29](https://user-images.githubusercontent.com/8849362/92002900-e89bec80-ed2f-11ea-8785-da62c273ba2a.png)

Reply:
Let me have a try

Reply:
Yes, it works perfectly.

![new.png](https://i.loli.net/2020/09/02/PzuKTeyGAsWY9SB.png)